UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -718,8 +718,20 @@ public void CmdRequestDisarm(GameObject playerToDisarm)
 		}
 	}
 
-	//admin only commands
+	[Command]
+	public void CmdGhostPerformTeleport(Vector3 s3, GameObject localPlayer)
+	{
+		ServerGhostPerformTeleport(s3, localPlayer);
+	}
+
+	[Server]
+	public void ServerGhostPerformTeleport(Vector3 s3, GameObject localPlayer)
+	{
+		Logger.Log("tried to teleport to " + s3);
```

Comment:
You're just going to spam useless logs on the server with this. I'd say remove it all together

Reply:
Yeah sorry missed that one